src/display/selection_draw.js

```diff
@@ -0,0 +1,139 @@
+import Pos from "../Pos";
```

Comment:
I think `display/selection.js` sufficiently suggests that this is about the way the selection is displayed (i.e. maybe leave off `_draw` in the filename)